scripts/plotting/polar_map.py

```diff
@@ -73,7 +73,12 @@ def polar_map(adfobj):
     # -- So check for it, and default to png
     basic_info_dict = adfobj.read_config_var("diag_basic_info")
     plot_type = basic_info_dict.get('plot_type', 'png')
-    print(f"NOTE: Plot type is set to {plot_type}")
+    print(f"\t NOTE: Plot type is set to {plot_type}")
+
+    # check if existing plots need to be redone
+    redo_plot = adfobj.get_basic_info('redo_plot')
+    print(f"\t NOTE: redo_plot is set to {redo_plot}")
```

Comment:
I might convert these print statements to debug log calls, e.g.:

```suggestion
    adfobj.debug_log(f"\t NOTE: Polar plot type is set to {plot_type}")

    # check if existing plots need to be redone
    redo_plot = adfobj.get_basic_info('redo_plot')
    adfobj.debug_log(f"\t NOTE: redo_plot is set to {redo_plot} for polar plots")
```